### PR TITLE
feat: add extension currency select page

### PR DESCRIPTION
**Pull request by emccorson**
This PR adds the currency select page to the extension.

### Expected behaviour:
- User can select a currency
- Selected currency persists after the page is reloaded
- Selecting a currency is not used anywhere to display a balance. This will be hooked up in a later PR.

Partially completes #675.

### CODE REVIEW

File: apps/extension/src/App/Settings/Currency.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import { GapPatterns, Stack, TickedRadioList } from "@namada/components";
+import { FiatCurrency } from "@namada/types";
+import { PageHeader } from "App/Common";
+
+import { ExtensionKVStore } from "@namada/storage";
+import { KVPrefix } from "router/types";
+import { LocalStorage } from "storage";
+import browser from "webextension-polyfill";
+
+const localStorage = new LocalStorage(
```
Comment (mateuszjasiuk) [inline apps/extension/src/App/Settings/Currency.tsx:12]: Not sure if we use this anywhere else in the extension, but we might make it a hook or sth :)

File: apps/extension/src/App/Settings/Currency.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  })
+);
+
+const FIAT_CURRENCIES = [
+  { code: "USD", symbol: "$" },
+  { code: "EUR", symbol: "€" },
+  { code: "GBP", symbol: "£" },
+  { code: "PLN", symbol: "zł" },
+  { code: "BRL", symbol: "R$" },
+  { code: "PHP", symbol: "₱" },
```
Comment (mateuszjasiuk) [inline apps/extension/src/App/Settings/Currency.tsx:25]: PHP best currency 😸

**Review by mateuszjasiuk — APPROVED** [review]
LGTM! It would be nice to add `cursor: pointer` on currencies hover :)

**Review by jurevans — APPROVED** [review]
Nice! This looks good

**Comment by emccorson** [comment]
This won't go in now that the extension requirements have changed.

**Pull request closed without merging**